README.md

```diff
@@ -123,28 +114,62 @@ This requires setting up [binfmt_misc](https://docs.kernel.org/admin-guide/binfm
 On most distributions, you can install QEMU user static to set up binfmt_misc handlers. For example, on Debian, use the command `apt install qemu-user-static`.
 
 
-## Customizing
-Building Garden Linux is based on a [feature system](features/README.md).
+### SELinux
 
-| Feature Type | Includes |
-|---|---|
-| Platforms | `ali`, `aws`, `azure`, `gcp`, `kvm`, `metal`, ... |
-| Features | `container host`, `virtual host`, ... |
-| Modifiers | `_slim`, `_readonly`, `_pxe`, `_iso`, ... |
-| Element | `cis`, `fedramp`, `gardener` |
+If you intend to build targets with the `_selinux` feature some additional requirements apply to the build machine.
```

Comment:
maybe here or in the podman section above an explicit mention about podman destkop might be good because for that enforcing selinux is the default, cf https://github.com/gardenlinux/builder/issues/16